### DISCUSSION TITLE
eHRAF Archaeology database

### DISCUSSION THREAD

**Discussion by rybesh** [periodo/periodo-data#95] [New period authorities]
The [traditions](https://ehrafarchaeology.yale.edu/ehrafa/browseCultures.do) of the eHRAF Archaeology database appear to fit the criteria for inclusion in PeriodO. At least some of them are adapted from other sources which could be included as well, e.g. the notes for [Highland Andean Archaic (SE47)](https://ehrafarchaeology.yale.edu/document?id=se47-000) say:

> The following major tradition summary has been adapted and compiled from "Early Highland Andean Archaic," by John Rick and "Late Highland Andean Archaic," by Lawrence Kuznar , from the Encyclopedia of Prehistory, Peter N. Peregrine and Melvin Ember, eds.

Thoughts @atomrab?

**Comment by atomrab** [comment]
Yes, I think these would fit, though this would put extra stress on the tension between periods as spans of time where some phenomenon/a is/are happening, and culture history, where the phenomena in question are archaeologically-identifiable peoples (this is what "archaeological traditions appears to mean in this context). That is, the "Highland Andean Archaic" is a timespan characterized by some sort of development, but "Inka" is both a culture and a span of time. 

I don't think we'd be able to do much with the derivedFrom relationship, because at least preliminary poking suggests that every entry is basically a reformatted version of an entry from Peregrine and Ember's Encyclopedia of Prehistory. I think we're just looking at the digital version of the "tradition" entries from that work, at least for the most part. 

I think this could probably be sorted out, but the bigger question is whether this is available in some sort of scrapable form (and what their licensing is, if so). I don't have the resources at the moment to translate all of these into PeriodO by hand, although I suppose I might be able to find an enthusiastic undergrad. I am a little worried, though, that this is meant to be a walled garden, and that they'd get bent out of shape if we attempted to model these traditions as PeriodO periods, especially since they seem to be at least partially membership-driven.
